app/controllers/application_controller.rb

```diff
@@ -31,15 +31,11 @@ def init_user
   end
 
   def notify(text, options = {})
-    notify_slack(text, options) if Rails.env.production?
-  end
-
-  def notify_slack(text, options = {})
-    webhook_url = "https://hooks.slack.com/services/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"
-    icon_url = options[:icon_url] || "http://i.gyazo.com/a8afa9d690ff4bbd87459709bbfe8be9.png"
+    return nil if Rails.env.production?
 
-    notifier = Slack::Notifier.new webhook_url
-    notifier.username = options[:username] || "256interns"
+    icon_url = options[:icon_url] || 'http://i.gyazo.com/a8afa9d690ff4bbd87459709bbfe8be9.png'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.